src/main/java/com/creativemd/littletiles/client/render/PreviewRenderer.java

```diff
@@ -195,27 +143,29 @@ public void tick(RenderHandEvent event) {
                     // Rotate Block
                     if (GameSettings.isKeyDown(LittleTilesClient.up) && !LittleTilesClient.pressedUp) {
                         LittleTilesClient.pressedUp = true;
-                        if (markedHit != null)
-                            moveMarkedHit(mc.thePlayer.isSneaking() ? ForgeDirection.UP : ForgeDirection.EAST);
+                        if (markedHit != null) moveMarkedHit(
+                                mc.thePlayer.isSneaking() ? ForgeDirection.UP : ForgeDirection.NORTH,
+                                direction_look);
                         else processKey(ForgeDirection.UP);
                     } else if (!GameSettings.isKeyDown(LittleTilesClient.up)) LittleTilesClient.pressedUp = false;
 
                     if (GameSettings.isKeyDown(LittleTilesClient.down) && !LittleTilesClient.pressedDown) {
                         LittleTilesClient.pressedDown = true;
-                        if (markedHit != null)
-                            moveMarkedHit(mc.thePlayer.isSneaking() ? ForgeDirection.DOWN : ForgeDirection.WEST);
+                        if (markedHit != null) moveMarkedHit(
+                                mc.thePlayer.isSneaking() ? ForgeDirection.DOWN : ForgeDirection.SOUTH,
+                                direction_look);
                         else processKey(ForgeDirection.DOWN);
                     } else if (!GameSettings.isKeyDown(LittleTilesClient.down)) LittleTilesClient.pressedDown = false;
 
                     if (GameSettings.isKeyDown(LittleTilesClient.right) && !LittleTilesClient.pressedRight) {
                         LittleTilesClient.pressedRight = true;
-                        if (markedHit != null) moveMarkedHit(ForgeDirection.SOUTH);
+                        if (markedHit != null) moveMarkedHit(ForgeDirection.EAST, direction_look);
                         else processKey(ForgeDirection.SOUTH);
```

Comment:
Why are these changing? Are the directions intentionally not aligning with the `else` cases?